app/models/video.rb

```diff
@@ -1,2 +1,26 @@
 class Video < ActiveRecord::Base
+  belongs_to :creator
+  validates_presence_of :uri, :embed, :vimeo_id, :posted_at
+  validates_uniqueness_of :uri
+
+  def self.create_videos_for_creator_from_hashes(video_data_array, creator)
+    if !video_data_array.nil?
```

Comment:
Instead of not nil (a double negative), I think you could use `if video_data_array` here! :tada: Double negatives can make code less readable.